### PR TITLE
API-6287 Remove requestingAdminEmail from change privacy policy and T&C events

### PR DESCRIPTION
**Pull request by petekirby-ee**

### CODE REVIEW

File: it/uk/gov/hmrc/thirdpartyapplication/repository/ApplicationRepositoryISpec.scala [hunk 1]
```diff
@@ -3006,7 +3006,7 @@ class ApplicationRepositoryISpec
       val app = anApplicationData(applicationId).copy(access = access)
       await(applicationRepository.save(app))
 
-      val event = ProductionAppPrivacyPolicyLocationChanged(UpdateApplicationEvent.Id.random, applicationId, LocalDateTime.now, gkUser, oldLocation, newLocation, adminEmail)
+      val event = ProductionAppPrivacyPolicyLocationChanged(UpdateApplicationEvent.Id.random, applicationId, LocalDateTime.now, gkUser, oldLocation, newLocation)
```
Comment (AndySpaven) [inline it/uk/gov/hmrc/thirdpartyapplication/repository/ApplicationRepositoryISpec.scala:3009]: I think this should use a collaborator actor (and several down below too)

Comment (petekirby-ee) [reply it/uk/gov/hmrc/thirdpartyapplication/repository/ApplicationRepositoryISpec.scala]: Fixed

File: test/uk/gov/hmrc/thirdpartyapplication/services/ApplicationUpdateServiceSpec.scala [hunk 2]
```diff
@@ -117,7 +117,7 @@ class ApplicationUpdateServiceSpec extends ApplicationUpdateServiceUtils
     val oldLocation           = PrivacyPolicyLocation.InDesktopSoftware
     val newLocation           = PrivacyPolicyLocation.Url("http://example.com")
     val changePrivacyPolicyLocation = ChangeProductionApplicationPrivacyPolicyLocation(instigator, timestamp, newLocation)
-    val event                       = ProductionAppPrivacyPolicyLocationChanged(UpdateApplicationEvent.Id.random, applicationId, timestamp, CollaboratorActor(adminEmail), oldLocation, newLocation, adminEmail)
+    val event                       = ProductionAppPrivacyPolicyLocationChanged(UpdateApplicationEvent.Id.random, applicationId, timestamp, CollaboratorActor(adminEmail), oldLocation, newLocation)
```
[the remark this answers is not in the document]
Comment (petekirby-ee) [reply test/uk/gov/hmrc/thirdpartyapplication/services/ApplicationUpdateServiceSpec.scala]: Fixed

File: test/uk/gov/hmrc/thirdpartyapplication/services/notifications/NotificationServiceSpec.scala [hunk 3]
```diff
@@ -78,7 +78,7 @@ class NotificationServiceSpec
       val previousPrivacyPolicyUrl = PrivacyPolicyLocation.Url("https://example.com/old-privacy-policy")
       val newPrivacyPolicyUrl = PrivacyPolicyLocation.Url("https://example.com/new-privacy-policy")
       val event = ProductionAppPrivacyPolicyLocationChanged(
-        UpdateApplicationEvent.Id.random, applicationId, LocalDateTime.now(), UpdateApplicationEvent.GatekeeperUserActor(gatekeeperUser), previousPrivacyPolicyUrl, newPrivacyPolicyUrl, adminEmail)
+        UpdateApplicationEvent.Id.random, applicationId, LocalDateTime.now(), UpdateApplicationEvent.CollaboratorActor(adminEmail), previousPrivacyPolicyUrl, newPrivacyPolicyUrl)
```
[the remark this answers is not in the document]
Comment (petekirby-ee) [reply test/uk/gov/hmrc/thirdpartyapplication/services/notifications/NotificationServiceSpec.scala]: Fixed

File: project/plugins.sbt [hunk 4]
```diff
@@ -3,9 +3,9 @@ resolvers += Resolver.url("HMRC-open-artefacts-ivy", url("https://open.artefacts
 
 resolvers += "Typesafe Releases" at "https://repo.typesafe.com/typesafe/releases/"
 
-addSbtPlugin("uk.gov.hmrc"       % "sbt-auto-build"        % "3.5.0")
+addSbtPlugin("uk.gov.hmrc"       % "sbt-auto-build"        % "3.8.0")
 addSbtPlugin("uk.gov.hmrc"       % "sbt-distributables"    % "2.1.0")
-addSbtPlugin("com.typesafe.play" % "sbt-plugin"            % "2.8.7")
+addSbtPlugin("com.typesafe.play" % "sbt-plugin"            % "2.8.8")
```
Comment (AndySpaven) [inline project/plugins.sbt:8]: Just check if this can be 2.8.15 - works in most places

**Review by AndySpaven — APPROVED** [review]

**Pull request merged**